src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -9,7 +9,7 @@
 // TODO: label all variables group - // * Menu toggle (vars) // - Menu toggle (selectors)
 
 @include pf-root($menu-toggle) {
-  --#{$menu-toggle}--ColumnGap: var(--pf-t--global--spacer--sm);
+  --#{$menu-toggle}--Gap: var(--pf-t--global--spacer--gap--text-to-element--default);
```

Comment:
Matches the button `--Gap` var, uses the correct semantic spacer, approved by design